File: samples/Samples/src/main/java/com/compdfkit/samples/util/CPDFGlideModule.java
package com.compdfkit.samples.util;

import com.bumptech.glide.annotation.GlideModule;
import com.bumptech.glide.module.AppGlideModule;

@GlideModule
public class CPDFGlideModule extends AppGlideModule {
    @Override
    public boolean isManifestParsingEnabled() {
        return false;
    }

}

File: samples/Samples_kotlin/src/main/java/com/compdfkit/samples/util/CPDFGlideModule.kt
package com.compdfkit.samples.util

import com.bumptech.glide.annotation.GlideModule
import com.bumptech.glide.module.AppGlideModule

@GlideModule
class CPDFGlideModule : AppGlideModule() {
    override fun isManifestParsingEnabled(): Boolean {
        return false
    }
}